rules/0020-syslog_rules.xml

```diff
@@ -523,6 +523,13 @@
     <group>pci_dss_10.2.5,pci_dss_10.2.2,gpg13_7.6,gpg13_7.8,gpg13_7.13,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,nist_800_53_AC.6,</group>
   </rule>
 
+  <rule id="5407" level="3">
+    <if_sid>5400</if_sid>
+    <regex> ; USER=\S+ ; COMMAND=| ; USER=\S+ ; TSID=\S+ ; COMMAND=</regex>
+    <description>Successful sudo executed </description>
```

Comment:
Remove the space at the end of the description. The sentence should finish with a dot.